gluon/tools.py

```diff
@@ -2950,7 +2945,7 @@ def login(self,
                                )
 
                 captcha = settings.login_captcha or \
-                    (settings.login_captcha != False and settings.captcha)
+                    (settings.login_captcha is notFalse and settings.captcha)
```

Comment:
Now fixed... Forget to git add tools.py :(

On Thu, Apr 14, 2016 at 11:05 AM, Richard Vézina <
ml.richard.vezina@gmail.com> wrote:

> Fixed
>
> On Thu, Apr 14, 2016 at 11:04 AM, Richard Vézina <
> ml.richard.vezina@gmail.com> wrote:
>
>> yes good catch...
>>
>> On Thu, Apr 14, 2016 at 11:02 AM, Cássio Botaro <notifications@github.com
>> > wrote:
>>
>>> In gluon/tools.py
>>> <https://github.com/web2py/web2py/pull/1296#discussion_r59733237>:
>>>
>>> > @@ -2950,7 +2945,7 @@ def login(self,
>>> >                                 )
>>> >
>>> >                  captcha = settings.login_captcha or \
>>> > -                    (settings.login_captcha != False and settings.captcha)
>>> > +                    (settings.login_captcha is notFalse and settings.captcha)
>>>
>>> i think you miss a space here
>>>
>>> —
>>> You are receiving this because you authored the thread.
>>> Reply to this email directly or view it on GitHub
>>> <https://github.com/web2py/web2py/pull/1296/files/4f26110ff40c521d150d55fedb952fbd65285111#r59733237>
>>>
>>
>>
>